nixos/modules/system/activation/switchable-system.nix

```diff
@@ -29,23 +25,17 @@ in
 
     enableNg = lib.mkOption {
```

Comment:
Not really important but worth mentioning: The most elegant option would probably be an `enum [ "perl" "rust" ]` similar to what we do for dbus https://github.com/NixOS/nixpkgs/blob/14efd10316e1c15a1139f8d22f898b7a0f797930/nixos/modules/services/system/dbus.nix#L41

Reply:
Yes, I agree that would have been ideal. But since this option shouldn’t be around forever anyway, I just did what I had to to maintain compatibility.